iree/compiler/Dialect/HAL/Target/VulkanSPIRV/VulkanSPIRVTarget.cpp

```diff
@@ -151,19 +254,41 @@ class VulkanSPIRVTargetBackend : public TargetBackend {
          executableOp.getBlock().getOps<IREE::HAL::ExecutableTargetOp>()) {
       if (matchPattern(executableTargetOp.target_backend(), name())) {
         ModuleOp innerModuleOp = executableTargetOp.getInnerModule();
-        if ((entryPointScheduleAttr = innerModuleOp.getAttrOfType<ArrayAttr>(
-                 iree_compiler::getEntryPointScheduleAttrName()))) {
-          auto spvModuleOps = innerModuleOp.getOps<spirv::ModuleOp>();
-          assert(llvm::hasSingleElement(spvModuleOps));
-          spvModuleOp = *spvModuleOps.begin();
-        }
+        auto spvModuleOps = innerModuleOp.getOps<spirv::ModuleOp>();
+        assert(llvm::hasSingleElement(spvModuleOps));
+        spvModuleOp = *spvModuleOps.begin();
+        entryPointScheduleAttr = innerModuleOp.getAttrOfType<ArrayAttr>(
+            iree_compiler::getEntryPointScheduleAttrName());
         break;
       }
     }
+    if (!spvModuleOp)
+      return executableOp.emitError("unable to find spv.module");
 
-    // If not found, then just fall back to the default dispatch recording.
+    SmallVector<spirv::FuncOp, 2> entryPointFns;
     if (!entryPointScheduleAttr) {
-      return TargetBackend::recordDispatch(loc, dispatchState, switchBuilder);
+      for (spirv::FuncOp spvFuncOp : spvModuleOp.getOps<spirv::FuncOp>())
```

Comment:
Use `{ ... }` to wrap around multi-line blocks.

Reply:
Done